mitmproxy/contentviews/img/png.ksy

```diff
@@ -0,0 +1,207 @@
+meta:
```

Comment:
I don't think we need this in the repo, a comment in the generated file where it comes from is sufficient.